tests/conftest.py

```diff
@@ -15,6 +17,10 @@
 # Early diagnostic for failed imports
 import pybind11_tests
 
+if os.name != "nt":
```

Comment:
nit Unclear why you can't allow Windows and use `spawn`?
(On that matter, why not just use `spawn` carte blanch?)

For testing, and given the size / speed of imports (small and fast!) for here, I doubt `forkserver` will give that much of an improvement?

Reply:
~9s vs ~13s

See added comment.

I think it's best / most future proof to not interfere under Windows: "If it's not broken, don't fix it."

Reply:
Gotcha, thanks for the info!